machine.go

```diff
@@ -85,6 +85,7 @@ const (
 	MachineTypeMicrovm string = "microvm"
 	MachineTypePC35    string = "q35"
 	MachineTypePC      string = "pc"
+	MachineTypeVirtAarch64 string = "virt"
```

Comment:
```suggestion
	MachineTypeVirt string = "virt"
```

I think it's fine to just call this MachineTypeVirt -- we can runtime check MachineType compat with host arch.